sync.go

```diff
@@ -55,10 +55,31 @@ type Group struct {
 	Keys []string `json:"keys"`
 }
 
+func decodeMemberList(body io.ReadCloser) (GoogleMemberList, error) {
```

Comment:
Why ReadCloser instead of io.Reader? You're not closing the body in these functions